### PR TITLE
Tutorial flags

### LINKED ISSUE #715: Support flags for rit tutorial
**What would you like to be added**:
Support flags for rit set credential

**Why is this needed**:
Improve integrability and standardize flags over stdin

### LINKED ISSUE #795: Tests with Testify in Pkg/RTutorial
**What would you like to be added**:
- [ ] Finder
- [ ] Setter

**Why is this needed**:

### PR DESCRIPTION
**Pull request by henriquemoraeszup**
### Description

* Adding flag support to `rit tutorial` command (closes #715 )
* Removing file manager dependencies on rtutorial package
* Refactoring `rtutorial` tests (closes #795 )
* Refactoring tests for cmd/tutorial command (contributes to #776 )

### How to verify it
You can run `rit tutorial --enabled=false` and check that the tutorial has been disabled

### Changelog

Added flag support for rit tutorial command

### CODE REVIEW

**Review by brunasilvazup — DISMISSED** [review]
LGTM :1st_place_medal:

**Review by lucasdittrichzup — DISMISSED** [review]

File: pkg/rtutorial/finder.go [hunk 1]
```diff
@@ -19,32 +19,28 @@ package rtutorial
...
 type FindManager struct {
 	tutorialFile string
-	homePath     string
-	fr           stream.FileReadExister
 }
 
-func NewFinder(homePath string, fr stream.FileReadExister) FindManager {
+func NewFinder(homePath string) FindManager {
 	return FindManager{
 		tutorialFile: fmt.Sprintf(TutorialPath, homePath),
```
Comment (kaduartur) [inline pkg/rtutorial/finder.go:32]: Could you use `filepath.Join()` to format the tutorial path?

File: pkg/rtutorial/setter.go [hunk 2]
```diff
@@ -19,35 +19,32 @@ package rtutorial
...
 
 type SetterManager struct {
 	tutorialFile string
-	fw           stream.FileWriter
 }
 
-func NewSetter(homePath string, fw stream.FileWriter) Setter {
+func NewSetter(homePath string) Setter {
 	return SetterManager{
 		tutorialFile: fmt.Sprintf(TutorialPath, homePath),
```
Comment (kaduartur) [inline pkg/rtutorial/setter.go:32]: Could you use `filepath.Join()` to format the tutorial path?

**Review by kaduartur — CHANGES_REQUESTED** [review]

**Review by kaduartur — APPROVED** [review]
LGTM 🚀

**Review by brunasilvazup — APPROVED** [review]

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**